packages/datadog-plugin-tedious/src/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+const Tags = require('opentracing').Tags
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+const tx = require('../../dd-trace/src/plugins/util/tx')
+
+function createWrapMakeRequest (tracer, config) {
+  return function wrapMakeRequest (makeRequest) {
+    return function makeRequestWithTrace (request) {
+      const connectionConfig = this.config
+      const scope = tracer.scope()
+      const childOf = scope.active()
+      const span = tracer.startSpan(`tedious.request`, {
+        childOf,
+        tags: {
+          [Tags.SPAN_KIND]: Tags.SPAN_KIND_RPC_CLIENT,
+          'db.type': 'mssql',
+          'service.name': config.service || `${tracer._service}-mssql`,
+          'resource.name': request.parametersByName.statement.value,
+          'span.type': 'tedious'
+        }
+      })
+
+      analyticsSampler.sample(span, config.analytics)
+      request.userCallback = tx.wrap(childOf, request.userCallback)
+      const res = scope.bind(makeRequest, span).apply(this, arguments)
```

Comment:
This returns a `Request` object so it should probably be called `request` instead of `res`.

Reply:
Nevermind, it doesn't return anything 👍